indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -68,6 +69,7 @@ def _process_POST(self):
             create_personal_data_fields(regform)
             db.session.add(regform)
             db.session.flush()
+            signals.registration_form_created.send(regform)
```

Comment:
yes, I see the signal added on line 72 and 98 in my branch... am I missing something?